Src/Particle/AMReX_Particle.H

```diff
@@ -25,6 +25,13 @@ namespace
     }
 
     using namespace LongParticleIds;
+
+    /** Flags used to set the entire uint64_t idcpu
+        to special values at once.
+     */
+    namespace ParticleIdCpus {
+        constexpr std::uint64_t Invalid = 16777216; // corresponds to id = -1, cpu = 0
+    }
```

Comment:
Adding this as well?
```suggestion
    }

    using namespace ParticleIdCpus;
```